scripts/approvalYears.py

```diff
@@ -978,18 +981,18 @@ def writeCBERBLAs(fpout, uniiPT, uniiALL):
             
     # write out all products data
     prod2UNIIs = dict()
-    for unii in UNII2prods.keys():
+    for unii in list(UNII2prods.keys()):
         for prod in UNII2prods[unii]:
-            if not prod2UNIIs.has_key(prod):
+            if prod not in prod2UNIIs:
                 prod2UNIIs[prod] = []
             prod2UNIIs[prod].append(unii)
 
     UNII2pt = dict()
 
-    for key in resolverCache.keys():
+    for key in list(resolverCache.keys()):
```

Comment:
Same as above.

Reply:
Python3 versus python2 issue. 2to3 automagically has made this change as .keys() no longer returns a list in python3